### PR TITLE
Update azure ad oidc docs

### PR DESCRIPTION
**Pull request by mooreds**
Fix for 

- https://github.com/FusionAuth/fusionauth-site/issues/156
- https://github.com/FusionAuth/fusionauth-issues/issues/855

### CODE REVIEW

**Comment by mooreds** [comment]
Wanted to merge this today or tomorrow @robotdan

File: site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc [hunk 1]
```diff
@@ -8,40 +8,31 @@ description: Login using Azure AD as an OpenID Connect Identity Provider
...
 == Configure OpenID Connect with Azure Active Directory
 
-[NOTE]
-====
-FusionAuth's OpenID Connect flow currently only supports link:https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-overview[Azure Active Directory v1.0].
-
-link:https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview[Azure Active Directory v2.0] returns inconsistent claims from the UserInfo endpoint depending on the type of Microsoft account the end-user has.  See link:https://github.com/MicrosoftDocs/azure-docs/issues/28317[this open issue] for reference.
-
-The email address is required to be returned on the Userinfo endpoint, without this identity claim FusionAuth cannot complete login. Currently the https://graph.microsoft.com/oidc/userinfo endpoint does not necessarily return the email address of the user even when requesting the `openid` and `email` scopes. These scopes do provide the `email` claim in the returned `access_token` and `id_token` but are omitted from the Userinfo response.
-====
```
Comment (robotdan) [inline site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc:18]: So is this document now correct for Azure AD v1 and v2? Or is there any differences or things we need to call out separately?

**Review by robotdan — COMMENTED** [review]
Looks good. 

While we are in here, we can re-capture the first screenshot (`/assets/img/docs/identity-providers/azure-openid-connect-login.png` to crop the bottom so it matches the others such as 

https://site-local.fusionauth.io/docs/v1/tech/identity-providers/google/
https://site-local.fusionauth.io/docs/v1/tech/identity-providers/twitter/

The other OpenID Connect examples all had this non-cropped bottom and we just haven't gotten back to them to fix.

File: site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc:18]: I wasn't able to figure out how to test with Azure AD1 (it's been deprecated) but confirmed by looking at the endpoints that my tests were going against Azure AD2. So it looks like the transition was seamless.

Comment (robotdan) [reply site/docs/v1/tech/identity-providers/openid-connect/azure-ad.adoc:18]: Ok, fair enough, so it sounds like we can forget about Azure AD1 - it is no longer in play.

**Pull request merged**